packages/server/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './http';
+export * from './http'; // dont export this from main?
```

Comment:
given that we now have taken the stance that http is not part of the trpc core, should we remove this export from the main?

I created a separate `@trpc/server/http` entrypoint

Reply:
i think that makes sense to me